core/src/main/java/com/yahoo/oak/NovaMMHeader.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2020, Verizon Media.
+ * Licensed under the terms of the Apache 2.0 license.
+ * Please see LICENSE file in the project root for terms.
+ */
+
+package com.yahoo.oak;
+
+import com.yahoo.oak.ValueUtils.ValueResult;
+
+import java.util.concurrent.atomic.AtomicLongArray;
+
+
+class NovaMMHeader {
+
+    /*
+    * Long NovaMMHeader: int version + int lock
+    * 0...  ...21 | 32...                  ...63
+    *  version    |  length
+    *
+    * The length (Mostly in integer size) of the data is also held just following the header (since the block and
```

Comment:
Unclear... Is the length part of the header or not? Is it that header keep only version and length. Where is the delete bit?